docs/Getting-Started/Starting-Pantheon.md

```diff
@@ -67,14 +67,13 @@ call [JSON-RPC API methods](../Reference/JSON-RPC-API-Methods.md) to confirm the
 To run a node that mines blocks at a rate suitable for testing purposes: 
 
 ```bash
-pantheon --dev-mode --network-id="2018" --bootnodes= --miner-enabled --miner-coinbase=0xfe3b557e8fb62b89f4916b721be55ceb828dbd73 --rpc-cors-origins="all" --ws-enabled --rpc-enabled --datadir=/tmp/tmpDatdir
+pantheon --dev-mode --network-id="2018" --bootnodes --miner-enabled --miner-coinbase=0xfe3b557e8fb62b89f4916b721be55ceb828dbd73 --rpc-cors-origins="all" --ws-enabled --rpc-enabled --datadir=/tmp/tmpDatdir
 ```
 
-Alternatively, use the following [configuration file](../Configuring-Pantheon/) to start a node with the same options as the above command line: 
+Alternatively, use the following [configuration file](../Configuring-Pantheon/) and `--bootnodes` on the command line to start a node with the same options as above: 
```

Comment:
This path is wrong and already is in the current doc version (leads to a 404 error page).
It should point to a .md file.